org.musicbrainz.Picard.json

```diff
@@ -16,9 +16,10 @@
         "--filesystem=xdg-music"
     ],
     "build-options": {
-        "cflags": "-O2 -g -pipe -fstack-protector-strong -D_FORTIFY_SOURCE=2",
-        "cxxflags": "-O2 -g -pipe -fstack-protector-strong -D_FORTIFY_SOURCE=2",
-        "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+        "cflags": "-O2 -g -fstack-protector-strong",
+        "cxxflags": "-O2 -g -fstack-protector-strong",
+        "cppflags": "-O2 -D_FORTIFY_SOURCE=2",
+        "ldflags": "-L/app/lib -fstack-protector-strong -Wl,-z,relro,-z,now",
```

Comment:
Won't we get that automatically when we move to FDO//18.08?

Reply:
Yes. But we're not yet. :)

Reply:
Let's just wait for that, then.